### DISCUSSION TITLE
求助，请问这是正版崩坏星穹铁道吗

### DISCUSSION THREAD

**Discussion by KnowPigeon** [share121/inter-knot#5275] [其他]
![55B7DFEF-1AB4-47FF-9E5B-377D944B9048](https://github.com/user-attachments/assets/dbe0c81f-ad24-4867-902e-af5b689174ef)
我是一个萌新，听说崩坏星穹铁道非常的好玩，于是我去下载了一个，请问这是正版吗<img src="https://img-static.mihoyo.com/communityweb/upload/3b2e51180d866548d11c650a2dcf6d43.png" alt="三月七_暗中观察" width="75" >

**Comment by zgy524911** [comment]
正的不能在正了

**Reply by KnowPigeon** [reply]
<img src="https://img-static.mihoyo.com/communityweb/upload/8f9225dcc57a028231f1d67fdab9135c.png" alt="三月七_骄傲" width="75" >

**Comment by manyshoes** [comment]
什么音游啊界面那么帅

**Reply by Yizakl** [reply]
原版是Phigros
这个看上去是自制铺qwq

**Reply by KnowPigeon** [reply]
事自制谱（
使用REPhityOS模拟器游玩
原版事Phigros（<img src="https://img-static.mihoyo.com/communityweb/upload/8debf4de875496c2515b005734200fe8.png" alt="三月七_吃糖" width="75" >

**Reply by manyshoes** [reply]
这就是屁股肉吗，<img src="https://bbs-static.miyoushe.com/static/2024/07/31/2dafa973b0fb29e5342403c4ead8af29_7443113916298582946.png" alt="三月七 比划" width="75" >，好牛

**Comment by momichibao** [comment]
phira

**Comment by nabeilishi** [comment]
有米莎，包正版的

**Reply by KnowPigeon** [reply]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by 210flight** [comment]
有logo，有人物🤔这就是崩铁🤓

**Reply by KnowPigeon** [reply]
<img src="https://img-static.mihoyo.com/communityweb/upload/8aee1ff2c6508b43813237163cacec1c.png" alt="三月七_加油" width="75" >
